### PR TITLE
Header

### PR DESCRIPTION
**Pull request by ruut975**

### CODE REVIEW

File: UI/Backdrop/Backdrop.module.css [hunk 1]
```diff
@@ -0,0 +1,9 @@
+.Backdrop {
+  width: 100%;
+  height: 100%;
+  position: fixed;
+  z-index: 100;
```
Comment (giograf) [inline UI/Backdrop/Backdrop.module.css:5]: This is good. I would also suggest using variables for z-index values like --global-z-index-backdrop. As an idea. Then when creating another overlapping element, you could see which value it should have based on the existing z-index variables (from the variable definition css file)

File: UI/Modal/Modal.module.css [hunk 2]
```diff
@@ -0,0 +1,20 @@
+.Modal {
+  position: fixed;
+  z-index: 500;
```
Comment (giograf) [inline UI/Modal/Modal.module.css:3]: Also possible variable use for z-index here.

File: components/NavItems/NavItems.module.css [hunk 3]
```diff
@@ -0,0 +1,63 @@
+.navItems {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  width: 100%;
+  background-color: var(--global-color-grey);
+  transition: transform 0.2s ease-in;
+  z-index: 200;
```
Comment (giograf) [inline components/NavItems/NavItems.module.css:8]: A variable opportunity.

File: components/NavItems/NavItems.module.css [hunk 4]
```diff
@@ -0,0 +1,63 @@
...
+
+.close {
+  transform: translateY(calc(-100% - 80px));
+}
+
+.inEnglish {
+  display: none;
+}
+
+/* desktop */
```
Comment (giograf) [inline components/NavItems/NavItems.module.css:24]: Desktop & Tablet

File: components/NavItems/NavItems.module.css [hunk 5]
```diff
@@ -0,0 +1,63 @@
...
+  .inEnglish {
+    display: inline-block;
+    position: fixed;
+    border: none;
+    background-color: var(--global-color-grey);
+    color: var(--global-color-white);
+    font-size: var(--global-font-size);
+    top: 7px;
+    right: 9px;
+    z-index: 500;
```
Comment (giograf) [inline components/NavItems/NavItems.module.css:43]: Z-index variable.

File: components/NavItems/NavItems.module.css [hunk 6]
```diff
@@ -0,0 +1,63 @@
...
+
+  .inEnglish:hover {
+    color: var(--global-color-vihr-primary-lightgreen);
+  }
+
+  .open {
+    transform: none;
+  }
+
+  .close {
```
Comment (giograf) [inline components/NavItems/NavItems.module.css:54]: `.open, .close` - you can combine the two css rules.

File: containers/Navbar/Navbar.js [hunk 7]
```diff
@@ -0,0 +1,47 @@
...
+
+  const menuToggleHandler = () => {
+    setShowMenu((prevState) => {
+      return !prevState;
+    });
+  };
+
+  let iconSrc = "/menu-icon.svg";
+  if (showMenu) {
+    iconSrc = "/x-mark-icon.svg";
```
Comment (giograf) [inline containers/Navbar/Navbar.js:23]: Here you could use one liner:
`iconSrc = showMenu ? ''one path"  : "another path"`

File: containers/Navbar/Navbar.js [hunk 8]
```diff
@@ -0,0 +1,47 @@
...
+  }
+
+  return (
+    <nav className={styles.navbarFlex}>
+      <div className={styles.wrapper}>
+        <a href="/">
+          <Logo />
+        </a>
+        <a href="#" className={styles.toggleNav}>
```
Comment (giograf) [inline containers/Navbar/Navbar.js:32]: You should use a button tag, because this is not a link.

File: containers/Navbar/Navbar.module.css [hunk 9]
```diff
@@ -0,0 +1,56 @@
+.navbarFlex {
+  display: flex;
+  flex-wrap: wrap;
+  justify-content: space-between;
+  align-items: center;
+  color: var(--global-color-white);
+  z-index: 300;
```
Comment (giograf) [inline containers/Navbar/Navbar.module.css:7]: Opportunity to use a variable.

File: containers/Navbar/Navbar.module.css [hunk 10]
```diff
@@ -0,0 +1,56 @@
...
+  font-size: var(--global-navlink-font-size);
+}
+
+/* desktop */
+@media (min-width: 768px) {
+  .navbarFlex {
+    flex-wrap: nowrap;
+    margin: 30px 20px 0 20px;
```
Comment (giograf) [inline containers/Navbar/Navbar.module.css:15]: `margin: 30px 20px 0`would suffice.

File: containers/Navbar/Navbar.module.css [hunk 11]
```diff
@@ -0,0 +1,56 @@
...
+  }
+}
+
+@media (min-width: 1024px) {
+  .navbarFlex {
```
Comment (giograf) [inline containers/Navbar/Navbar.module.css:20]: Can be removed

File: containers/Navbar/Navbar.module.css [hunk 12]
```diff
@@ -0,0 +1,56 @@
...
+@media (min-width: 768px) {
+  .wrapper {
+    flex: 1 1 400px;
+    justify-content: space-around;
+    margin: 0;
+    margin-bottom: 3px;
+  }
+}
+
+@media (min-width: 1024px) {
```
Comment (giograf) [inline containers/Navbar/Navbar.module.css:41]: Can be removed.

File: containers/Navbar/Navbar.module.css [hunk 13]
```diff
@@ -0,0 +1,56 @@
...
+  .wrapper {
+  }
+}
+
+.toggleNav {
+  margin: 6px 9px 0 9px;
```
Comment (giograf) [inline containers/Navbar/Navbar.module.css:47]: `margin: 6px 9px 0`

File: containers/Navbar/Navbar.module.css [hunk 14]
```diff
@@ -0,0 +1,56 @@
...
+  z-index: 500;
```
Comment (giograf) [inline containers/Navbar/Navbar.module.css:48]: Z-index variable

**Review by giograf — CHANGES_REQUESTED** [review]

File: components/NavItems/NavItems.module.css (hunk 6 above)
Comment (ruut975) [reply components/NavItems/NavItems.module.css:54]: ok

File: components/NavItems/NavItems.js [hunk 15]
```diff
@@ -0,0 +1,27 @@
...
+  return (
+    <>
+      <Backdrop show={props.open} clicked={props.closed} />
+      <div className={attachedClasses.join(" ")} onClick={props.closed}>
+        <NavItem path={"/paula"}>Paula</NavItem>
+        <NavItem path={"/teemat"}>Teemat</NavItem>
+        <NavItem path={"/kuntavaalit-2021"}>Kuntavaalit 2021</NavItem>
+        <NavItem path={"/blogi"}>Blogi</NavItem>
+        {props.open && <NavItem path={"/eng"}>In English</NavItem>}
+
```
Comment (giograf) [inline components/NavItems/NavItems.js:20]: Remove empty line.

**Pull request merged**